### PR TITLE
Car widget structure

### PR DESCRIPTION
**Pull request by Infinityworld13**
Added and moved the Car Widget structure to the Client Banking page folder

## Context

The current Client Banking page had no way to show/output the list of cars generated through the backend. This is related to issue #21 

## Your Changes

**Description**:

Made carSlide construct a CustomCard architecture where we can show individual car objects.
Made Widget build a Caroussel class that stores the carSlides and enables intractability with the user to slide and see more of them.
Their folder Widget was moved into the Mock Banking Page folder instead of components so that it could be utilized in the index.js

**Type of change** (select all that apply):

- [x] New feature (additional functionality separate from existing code)
- [x] Refactoring (changing code structure only)
- [ ] Breaking change (changing code functionality)
- [ ] Documentation update (adding more comments)
- [ ] Bug fix (non-breaking change that solves an error/issue)
- [ ] User Interface change (design changes)
- [ ] Test update (adding test code)
- [ ] Other (please specify):

## Checklist

- [x] Code is up to date with main branch (no merge conflicts)
- [x] Code has been properly tested
- [x] Code has been properly documented
- [ ] One or two people have reviewed this pull request

### CODE REVIEW

File: mock banking page/index.html [hunk 1]
```diff
@@ -51,6 +51,7 @@ <h2>Borrowing</h2>
                 </tr>
             </table>
         </div>
+        <div id="widget-container"></div>
```
Comment (leenlab2) [inline mock banking page/index.html:54]: why did you move it?

File: mock banking page/widget/carousel.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ * The user can interact with it through the arrows or by slidling their cursor to obtain the next card.
+ *
+ * It takes as props the values for each card
+ */
+class Carousels extends React.Component {
+    constructor(props){
+        super()
+        this.props = props;
+    }
+    renderItems() {
```
Comment (leenlab2) [inline mock banking page/widget/carousel.js:18]: can you add documentation to the methods?

File: mock banking page/index.html (hunk 1 above)
Comment (leenlab2) [reply mock banking page/index.html:54]: include documentation that this is a temporary change

**Review by leenlab2 — COMMENTED** [review]
good progress. left some minor comments.

File: mock banking page/widget/carousel.js (hunk 2 above)
Comment (Infinityworld13) [reply mock banking page/widget/carousel.js:24]: Done

File: mock banking page/index.html (hunk 1 above)
Comment (Infinityworld13) [reply mock banking page/index.html:56]: Done

**Review by nguya03584 — APPROVED** [review]

**Comment by leenlab2** [comment]
[Implement Widget on IntelliJ](https://trello.com/c/gpmyRnUq/108-implement-widget-on-intellij)
